### ISSUE TITLE
Unhandled 'error' event

### ISSUE THREAD

**Issue by rudyportillo101** [tnicola/cypress-parallel#11]
**cypress-parallel:** 0.1.7
**cypress:** 4.12.1
**NPM:** 6.14.4
**OS:** macOS

```
Creating weight file: cypress/parallel-weights.json
events.js:292
      throw er; // Unhandled 'error' event
      ^

Error: spawn yarn ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:267:19)
    at onErrorNT (internal/child_process.js:467:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:273:12)
    at onErrorNT (internal/child_process.js:467:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'spawn yarn',
  path: 'yarn',
  spawnargs: [
    'run',
    'cy:run',
    '--',
    '--reporter',
    'cypress-parallel/json-stream.reporter.js',
    '--spec',
    "'cypress/integration/audit-lighthouse.spec.js,cypress/integration/cookies.spec.js,cypress/integration/notification.spec.js,cypress/integration/visual.spec.js'"
  ]
}
```

**Comment by tnicola** [comment]
@rudyportillo101 Try with the new 0.1.8 version.

**Comment by rudyportillo101** [comment]
Thanks @tnicola for your response, I updated to version 0.1.8, but I'm getting this error:

`Error: listen EADDRINUSE: address already in use :::5338`

**Comment by rudyportillo101** [comment]
@tnicola the issue is fixed in version 0.1.9. Thanks!
